README.md

```diff
@@ -2,3 +2,5 @@
 Just the classic hello world project to get used to repositories, branches, commits, and pull requests!
 
 Well, hello world!  I am currently making changes to my readme file.  Changes on GitHub are called committs. 
+
+This is more practice...trying to see how the @mention system works. i will me @'ing my bff MH (m-hintz-42)  
```

Comment:
Not bad but I think `@` is used as a link or specific attribute in `.md` (README) files.

Reply:
Thank you for your review and comment.